setup.py

```diff
@@ -21,15 +21,16 @@
     },
     packages=['vat'],
     install_requires=['requests', 'pytz', 'cabby', 'stix'],
-    python_requires='>=2.6, !=3.0.*, !=3.1.*, !=3.2.*, <4',
+    python_requires='>=3.6',
```

Comment:
ubuntu 12.04 shipping with python 3.4 and ubuntu 16.04 shipped with 3.5.  I would recommend requiring >= to 3.4 to support running on older OS releases.